en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md

```diff
@@ -604,7 +604,7 @@ file, under the same directory as your `main.md`. Let's see if this
 works. Save your file, switch to the terminal window and run:
 
 ```
-$ pandoc main.docx --filter pandoc-citeproc -o main.md
+$ pandoc main.docx --filter pandoc-citeproc -o main.docx
```

Comment:
This command is incorrect. It should be: `$ pandoc main.md --filter pandoc-citeproc -o main.docx`